### PR TITLE
Deploy Portfolio in github page

### PR DESCRIPTION
**Pull request by stephenkati**
- Push well working version of the app to production
- Include online link to the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ write the exact location of the file on your browser and choose index.html file
 
 ### Run tests
 
-To run tests, run the following command:
+The app is deployed on Github pages. Follow the link below: https://stephenkati.github.io/Portfolio-setup-and-mobile-first/
```
Comment (ha-manel) [inline README.md]: - [ ] Kindly add the link in the `live demo` section, your live demo section says that the website is not deployed yet 👍

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @stephenkati 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* The website is deployed correctly 💯 

## Required Changes:

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -115,7 +115,7 @@ A list of relevant skills: List the programming languages, frameworks, and tools
 
 > Add a link to your deployed project.
```
Comment (Ibizugbe) [inline README.md:116]: [OPTIONAL] I think you can do away with this line. it is making your readme still look like a template😉

**Review by Ibizugbe — APPROVED** [review]
Hi @stephenkati 👋, 

BigTvGIF (3)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅
- [x] Readme file contains live demo link👍
- [x] Github pages has been used for deployment👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**